### PR TITLE
xml children module parameter does not exist

### PR DESCRIPTION
**Pull request by Warkdev**
##### SUMMARY
Fixes https://github.com/ansible-collections/community.general/issues/4166

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
xml

### CODE REVIEW

**Comment by Warkdev** [comment]
Issue reporter: [ooola](https://github.com/jooola)

**Review by russoz — APPROVED** [review]
LGTM

File: changelogs/fragments/5808-xml-children-parameter-does-not-exist.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+- xml - fixed a bug where empty ``children`` list would not be set (https://github.com/ansible-collections/community.general/pull/5808)
```
Comment (felixfontein) [inline changelogs/fragments/5808-xml-children-parameter-does-not-exist.yml:2]:
```suggestion
- xml - fixed a bug where empty ``children`` list would not be set (https://github.com/ansible-collections/community.general/pull/5808).
```

**Review by felixfontein — COMMENTED** [review]
Thanks for your contribution!

**Comment by felixfontein** [comment]
@Warkdev thanks for your contribution!
@russoz thanks for reviewing!

**Pull request merged**